app/src/main/res/layout/app_bar.xml

```diff
@@ -79,15 +79,15 @@
         <View
             android:layout_width="1dp"
             android:layout_height="match_parent"
-            android:background="?attr/colorPrimary" />
+            android:background="?attr/white" />
 
         <ImageButton
             android:id="@+id/action_big_owl"
             android:layout_width="0dp"
             android:layout_height="match_parent"
             android:layout_margin="10dp"
             android:layout_weight="1"
-            android:background="@color/white"
+            android:background="@color/colorPrimary"
```

Comment:
This parameter value will make the background to be blue whereas the other icons background are white. To have the same white color for all  icons background you should leave it as unchanged